### PR TITLE
To do list

### PR DESCRIPTION
**Pull request by Osoro254Alex**
I did the following on this Repo:

- Created Repo To-Do-List.
- Cloned the Repo.
- Created a feature branch.
- Modify the README.md file.
- Initialized my repo.
- Install webpack.
- Add .gitignore file.
- Add src/ folder with .html, .css, and .js files.
- Add dist/ folder.
- Configure the webpack.
-  Fix Linter errors.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  {
+    description: 'complete the To Do list project',
+    completed: false,
+    index: 3,
+  },
+];
+
+const loadList = () => {
+  const listCont = document.querySelector('.list-cont');
+
```
Comment (gemmen29) [inline src/index.js]: - [ ] Kindly sort the items list based on their index value before you display it to achieve this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=On%20page%20load%20render%20the%20dynamically%20created%20list%20of%20tasks%20in%20the%20dedicated%20placeholder.%20The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Osoro254Alex, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Osoro254Alex** [comment]
@gemmen29 
Sort the items list based on their index values.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Osoro254Alex 

Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- array well sorted
- structure well put in place
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Osoro254Alex** [comment]
@tufoinnkuo10 

- Checked and Confirmed I did changes as per the previous reviewer.
- Approved.

**Pull request merged**